beater/route_config.go

```diff
@@ -0,0 +1,152 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package beater
+
+import (
+	"net/http"
+	"regexp"
+
+	perr "github.com/elastic/apm-server/processor/error"
+	"github.com/elastic/apm-server/processor/metric"
+	"github.com/elastic/apm-server/processor/sourcemap"
+	"github.com/elastic/apm-server/processor/transaction"
+
+	"github.com/elastic/apm-server/decoder"
+	"github.com/elastic/apm-server/transform"
+	"github.com/elastic/beats/libbeat/logp"
+)
+
+var (
+	rootURL                   = "/"
+	BackendTransactionsURL    = "/v1/transactions"
```

Comment:
thanks @simitt. You mention that you're not in favour of introducing _backend_. I'm not really _introducing_ the terminology here, just moved the definitions and and then used the same terminology for v2. 

I'm not sure if you're suggesting i just change the v2 terminology or i change all of them, but in any case, i would prefer to to not change terminology for v1 in this PR because i don't think that terminology relates to the changes here. And if we're not changing v1 terminology here, then i don't think we should use a different terminology for v2 because that would make it inconsistent.